src/ucp/wireup/select.c

```diff
@@ -1880,7 +1881,7 @@ ucp_wireup_select_wireup_msg_lane(ucp_worker_h worker,
     ucp_lane_index_t lane;
     unsigned addr_index;
 
-    ucp_wireup_fill_aux_criteria(&criteria, ep_init_flags);
+    ucp_wireup_fill_aux_criteria(&criteria, ep_init_flags, 0);
```

Comment:
we need CB_ASYNC here

Reply:
@yosefe Should it add ```CB_ASYNC``` here?
For the ```wireup msg lane```, the ```ep_init_flags``` have the same flags as CM lane. In the original code logic, it doesn't require the CB_ASYNC capability.
If it adds ```CB_ASYNC``` here, the function [ucp_wireup_select_wireup_msg_lane](https://github.com/openucx/ucx/blob/v1.15.0-rc1/src/ucp/wireup/select.c#L1883) may have different execution code path compared with the original code logic.

Reply:
Please ignore the above comment after reading below comment.
https://github.com/openucx/ucx/pull/9061#discussion_r1193545512

Reply:
Yes, in order to keep the original logic